data/loot_lists/level1/level1_accessories.yml

```diff
@@ -15,10 +15,12 @@ weighted_entries:
       warlock_power1: 2
       mage_ability1: 2
       ability1: 2
+    - identified: 90

```

Comment:
Shouldn't we just be using "none" here?  do we need the separate "identified" item adjective?